WGLMakie/src/WGLMakie.jl

```diff
@@ -80,7 +80,7 @@ function __init__()
     atlas = wgl_texture_atlas()
     TEXTURE_ATLAS[] = convert(Vector{Float32}, vec(atlas.data))
     Makie.font_render_callback!(atlas) do sd, uv
-        TEXTURE_ATLAS[] = convert(Vector{Float32}, vec(wgl_texture_atlas().data))
+        TEXTURE_ATLAS[] = convert(Vector{Float32}, vec(sd))
```

Comment:
This is wrong, since sd is just the signed distancefield of the newly rendered glyph.
So this _should_ update the texture atlas observable in the browser.
I suspect some race condition, that the plot is rendered first before all glyphs are updated in the texture atlas.